blocks/library/embed/index.js

```diff
@@ -22,6 +26,23 @@ registerBlock( 'core/embed', {
 	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, title, caption } = attributes;
 
+		if ( ! url ) {
+			return (
+				<div className="blocks-embed is-placeholder">
```

Comment:
> This sounds like a <Placeholder> reusable component. @jasmussen do we expect the design to share these commonalities between image/embed/video/etc placeholders?

Yes, the design for a placeholder block is always 90% the same. Always a block, always an icon and a title centered, and always some more options after that. Those more options are likely going to be different, as @BE-Webdesign suggests:

- Image will have a "open media library" button
- Embeds will have a URL option
- We'll probably run into other things as well.

So maybe something like, and please correct my pseudo code:

```
<Placeholder icon="cloud" title="Embed>
... input field and button ...
</Placeholder>
```